ui/packages/diagnosis_report/src/types.ts

```diff
@@ -0,0 +1,21 @@
+import { createContext } from 'react'
+
+export interface TableRowDef {
+  Values: string[]
+  SubValues: string[][]
+  ratio: number
+  Comment: string
+}
+
+export interface TableDef {
+  Category: string[]
+  Title: string
+  CommentEN: string
+  CommentCN: string
+  joinColumns: number[]
```

Comment:
why some are UpperCased while some are lowerCased?

Reply:
just copied from go code ...

Reply:
@crazycs520 We should fix it..